From: ipsec-tools packagers <team+ipsec-tools@tracker.debian.org>
Date: Sun, 7 Jun 2020 15:41:18 +0200
Subject: move-private-libraries-to-subfolder

this patch is setting the rpath in ltmain.sh to move the private libraries
afterwards in a subfolder under /usr/lib/ to follow the recommendations of
debian policy 10.2
---
 ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ltmain.sh b/ltmain.sh
index 63ae69d..c589d0d 100755
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -5118,7 +5118,7 @@ func_mode_link ()
     finalize_command=$nonopt
 
     compile_rpath=
-    finalize_rpath=
+    finalize_rpath=/usr/lib/ipsec-tools
     compile_shlibpath=
     finalize_shlibpath=
     convenience=
